controller/auth_utility_test.go

```diff
@@ -22,13 +22,14 @@ import (
 
 const (
 	// the various HTTP endpoints
+	endpointWorkItem               = "/api/workitems"
```

Comment:
the goal of this PR is to remove the `/spaces?<space_id>` fragment in the URL when requesting a specific work item. There's actually no need to pass the id of the space since the wok item ID is unique.